cmd/kube-controller-manager/app/core.go

```diff
@@ -79,8 +79,10 @@ func startServiceController(ctx ControllerContext) (http.Handler, bool, error) {
 }
 
 func startNodeIpamController(ctx ControllerContext) (http.Handler, bool, error) {
-	var clusterCIDR *net.IPNet = nil
-	var serviceCIDR *net.IPNet = nil
+	var (
+		clusterCIDR *net.IPNet = nil
```

Comment:
While you're at it, no need to assign `= nil` here as it is the default value, right?

Reply:
+1.

@liangwenguo thanks for the clean up!